torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3597,13 +3597,13 @@ def sample_inputs_log_softmax(op_info, device, dtype, requires_grad, with_dtype=
     make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
 
     if with_dtype:
-        cases = (((S, S, S), (1, torch.float64)),)
+        cases = (((S, S, S), (1, ), {'dtype': torch.float64}),)
```

Comment:
Had to make `dtype` as kwarg because `nn.functional.log_softmax` has a positional argument `_stacklevel` before `dtype`.

https://pytorch.org/docs/stable/generated/torch.nn.functional.log_softmax.html#torch.nn.functional.log_softmax